tests/introspection/Mac/MacApiCtorInitTest.cs

```diff
@@ -68,6 +68,10 @@ protected override bool Skip (Type type)
 			// [NSWindow initWithContentRect:styleMask:backing:defer]
 			case "MonoMac.AppKit.NSWindow":
 			case "AppKit.NSWindow":
+			// *** -[NSKeyedArchiver dealloc]: warning: NSKeyedArchiver deallocated without having had -finishEncoding called on it.
+			case "MonoMac.Foundation.NSKeyedArchiver":
+			case "Foundation.NSKeyedArchiver":
```

Comment:
It loks like it is just a warning, the test itself passes.  Is there a way to ignore the warning?